app/facades/food_facade.rb

```diff
@@ -0,0 +1,12 @@
+class FoodFacade
+
+  def self.foods_search(query)
+    FoodService.foods_search(query)[:foods].map do |data|
+      Food.new(data)
+    end
+  end
```

Comment:
I can see a world where this method does more heavy lifting, returning `totalHits` as well as the Food POROs:
```
def self.foods_search(query)
  food_data = FoodService.foods_search(query)
  foods = food_data.first(10).map do |data|
    Food.new(data)
  end
  [food_data[:totalHits], foods]
end
```

With this implementation, you would not need the `SearchInfo` PORO or the `search_info` method below. I don't necessarily think that having a `SearchInfo` PORO is a poor design choice, but there is the drawback of hitting the same endpoint more times than is needed for the requested behavior.